src/index.html

```diff
@@ -9,15 +9,17 @@
       html {
         --color-text-primary: #2a3443;
         --color-text-secondary: #5e6673;
-        --color-background: #db960a;
+        --color-background: #221f18;
         --color-light-grey: #f4f3f2;
 
         --color-background-all: #6d5f47;
         --color-background-light: #ffeed4;
         --color-footer-bar: #221f19;
         --color-footer-bar-content: #302f2c;
 
-        --color-sidebar-menu: #a79570;
+        --color-sidebar-menu-old: #a79570; 
```

Comment:
If this is not used any more, best to remove it (I checked and it is not used now)

Reply:
I will remove this one: it is always still there in the GitHub history if you need it.

Reply:
is removed